test/cli-alert/src/index.ts

```diff
@@ -202,6 +213,21 @@ async function run() {
       await waitForConclusion(id);
     }
 
+    console.log('Checking smoke tests jobs...');
+    const failedWorkflows = await discoverConsecutiveFailures(
+      latestWorkflowRuns[0].id,
+      latestWorkflowRuns[1].id,
+    );
+
+    if (!failedWorkflows.length || failedWorkflows.length < 1) {
```

Comment:
`failedWorkflows.length < 1` would mean that `failedWorkflows` is `0`, and `!!0 = false` so it's already checked by the first condition.

```suggestion
    if (!failedWorkflows.length) {
```